client/components/forms/form-text-input/index.jsx

```diff
@@ -40,7 +43,7 @@ export default class FormTextInput extends PureComponent {
 			<input
 				type="text"
 				{ ...props }
-				ref="textField"
+				ref={ inputRef || 'textField' }
```

Comment:
Sort of?

As written, if you pass an `inputRef` and then call `focus()` on the component you'll dereference a `null`.

From the parent's perspective, setting `ref="child"` and `inputRef` is legitimate - the child will be available in `parent.refs.child`.

We could support that if we wanted to if we switched to using callbacks. Something like:
`const wrappedRef = ( el ) => ( this.ref = el, inputRef( el )`.

I've not implemented that mainly because:

- it seems like supporting wrongness
- The pattern of passing `inputRef` down to the terminal input components seems a lot nicer to me
- It seems a little bit "clever"

I'm happy to put it in if you think it's appropriate (or if something explodes).